roles/redis/defaults/main.yml

```diff
@@ -0,0 +1 @@
+redis_image: quay.io/akumari/redis-rpm:nightly
```

Comment:
This depends on the outcome of https://github.com/theforeman/foreman-quadlet/pull/97